ui/config/proxy.ts

```diff
@@ -5,24 +5,29 @@
  * For details, please see
  * https://pro.ant.design/docs/deploy
  */
+const { PROXY_PORT } = process.env
```

Comment:
set a default port?

Reply:
ok  have changed to `process.env.PROXY_PORT || 8000`